framework/UPGRADE.md

```diff
@@ -15,6 +15,14 @@ php composer.phar self-update
 php composer.phar global require "fxp/composer-asset-plugin:^1.2.0"
 ```
 
+Upgrade from Yii 2.0.9
+----------------------
+
+* MSSQL: NTEXT data type marked as deprecated. (https://msdn.microsoft.com/en-us/library/ms187993.aspx) 
```

Comment:
`NTEXT` data type was marked as deprecated (https://msdn.microsoft.com/en-us/library/ms187993.aspx) so `\yii\db\Schema::TYPE_TEXT` was changed from `ntext` to `nvarchar`. By default `nvarchar` defines a string of length 4000. If you need `Schema::TYPE_TEXT = nvarchar(max)`, you can adjust it in your code. It's not like that by default because `MAX` keyword is new to SQL Server 2005.